docs/rules/jsx-no-duplicate-props.md

```diff
@@ -1,6 +1,8 @@
-# Disallow duplicate properties in JSX (react/jsx-no-duplicate-props)
+# Disallow duplicate properties in JSX (`react/jsx-no-duplicate-props`)
 
-💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations): `all`, `recommended`.
+✅ This rule is enabled in the `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/README.md#shareable-configs).
```

Comment:
can we change the checkmark here? "recommended" turns out to be a poor name, and i want to minimize endorsement of it.

Reply:
I solved this by passing the `--config-emoji recommended,💼` option to use the generic emoji for this config.